test/specs/testnet/swap/Regression_SingleChainSwap.spec.js

```diff
@@ -538,4 +538,355 @@ describe("The regression suite for the single chain swap on the TestNet", () =>
       );
     }
   });
+
+  // SWAP ON XDAI NETWORK FROM ERC20 TOKEN TO ERC20 TOKEN WITH EXCEED TOKEN BALANCE
+  it("Setup the SDK for xDai network and perform the single chain swap action from ERC20 token to ERC20 Token with exceed token balance.", async () => {
+    // Initialize the SDK and define network
+    sdkTestNet = new Sdk(process.env.PRIVATE_KEY, {
```

Comment:
Remember to name these SDK instance variables properly

Reply:
Updated the SDK instance name in all the files.